src/renderer/store/slices/device/validation/pins.ts

```
@@ -0,0 +1,183 @@
+ *    - Following the prefix, the address must have a integer number starting with 0
+ *    - Following the number, the address must have a dot "."
+ *    - Following the dot, the address must have a integer number starting with 0 and ending with 7
+ * 2. For analog types:
+ *    - The address must start with the prefix "%QW" or "%IW"
+ *    - Following the prefix, the address must have a integer number starting with 0
+ */
+const pinAddressValidation = (pinAddress: string, pinType: string) => {
+  switch (pinType) {
+    case 'digital':
+      return addressPrefixRegex.digital.test(pinAddress) && pinAddress.split('.')[1] <= '7'
+      break
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)